README.md

```diff
@@ -44,40 +51,36 @@ The project is designed to run on your local computer, provided you have met the
 }
 ```
 
-### Using Azure Functions Core Tools (CLI)
+### Using Functions (CLI)
 
 1) Open a new terminal and do the following:
 
 ```bash
-cd FunctionHttp
 func start
 ```
 
-2) Test a Web hook or GET using the browser to open http://localhost:7071/api/httpget
+2) Test the HTTP GET trigger using the browser to open http://localhost:7071/api/httpget
 
-3) Test a POST using your favorite REST client, e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client), PostMan, curl. `test.http` has been provided to run this quickly.
+3) Test the HTTP POST trigger using your favorite REST client (e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)). `test.http` has been provided to run this quickly.
 Or in a new terminal run the following:
 
 ```bash
-curl -i -X POST http://localhost:7071/api/httppost \
-  -H "Content-Type: text/json" \
-  --data-binary  "@testdata.json"
+curl -i -X POST http://localhost:7071/api/httppost -H "Content-Type: text/json" -d "{\"name\": \"Developer\"}"
```

Comment:
Works, but why not exact same object as the others (and escape in chars of course)?

Using this object:
```json
{
  "name": "Awesome Developer",
  "age": 25
}
```

The curl changes to

```suggestion
curl -i -X POST http://localhost:7071/api/httppost -H "Content-Type: text/json" -d "{\"name\": \"Awesome Developer\", \"age\": \"25\"}"
```

Reply:
fixed